Source/RunActivity/Viewer3D/Debugging/DebugViewerForm.cs

```diff
@@ -1937,6 +1937,7 @@ private void boxSetSignalChosen(object sender, EventArgs e)
                       else if (drawstate2 > 0) { sigHead.state = MstsSignalAspect.APPROACH_2; }
                       else { sigHead.state = MstsSignalAspect.APPROACH_3; }
                       sigHead.draw_state = sigHead.def_draw_state(sigHead.state);
+					  sigHead.TextSignalAspect = "";
```

Comment:
Sorry, but what does this line too?

Reply:
The way the dispatcher window is implemented bypasses the signal logic and forces a fixed aspect. In order to not confuse other signals and TCS scripts when an (msts)aspect is forced, the text aspect has to be reset, otherwise it would be kept in an inconsistent state.

Reply:
Thanks, that clears things up!